### ISSUE TITLE
Ubertooth

### ISSUE THREAD

**Issue by NoseMith** [curtbraz/WhosHere#3]
What a great tool! Any plans for bluetooth support  as well?  Like Blue_Hydra?
Again, great job!

**Comment by curtbraz** [comment]
Thanks!  Yes, ironically, I have an UberTooth and have been experimenting with it to see how I could leverage it with WhosHere.  I have a plan for implementation just need to find the free time to develop. :)  I'm hoping I'll have support for it within the next 6 mo or so.  I want to first bypass the MAC randomization and then Bluetooth is next up.

**Comment by alfem** [comment]
I have read somewhere MAC randomization can be 'defeated' using 'capabilites' info in wireless packets. 

Unfortunately I can not find any opensource sample code (only commercial products)

**Comment by curtbraz** [comment]
Hey there!  Back when I researched it I believe I read a simple RST packet
should do the trick, but I need to brush up again and implement the fix.

**Comment by That-Dude** [comment]
It would be great if you could implement the RST packet trick

**Comment by curtbraz** [comment]
Yes, I really want to.  I'm very sorry for my lack of involvement.. it's
difficult to find time with my day job and all of my other projects but
this is something I want to completely revamp and improve soon.  I might
just have to take a week of PTO. :)

**Comment by That-Dude** [comment]
Do it :-)

I would help if I wasn’t such an awful coder. I can just about cobble things together in bash. But if you need photography I’m on it!

**Comment by curtbraz** [comment]
Haha well I'm a terrible coder myself but I'll see what I can do! :)
